corporate/lib/analytics.py

```diff
@@ -39,3 +43,37 @@ def estimate_annual_recurring_revenue_by_realm() -> Dict[str, int]:  # nocoverag
             # TODO: Decimal stuff
             annual_revenue[plan.customer.realm.string_id] = int(renewal_cents / 100)
     return annual_revenue
+
+
+def estimate_annual_recurring_revenue_by_remote_server() -> Dict[int, int]:  # nocoverage
+    annual_revenue: Dict[int, int] = {}
```

Comment:
Probably you want this to may `int` to a `RemoteServerPlanData` `dataclass` with the a `plan_type`, `plan_name`, and ARR; that'll make it easy to add other fields about the plan is relevant without rethinking the plumbing much.

Reply:
What do we want to do for servers with multiple active plans (e.g. multiple realms)?

Reply:
I think ultimately we'll want to adjust how we display things but for now, maybe just display the set of plan names separated by a comma or something? That seems easy enough.

Reply:
Talked with @alya and for now when there are multiple plans for a remote server, the current plan name and status columns will send set strings to send the admin to the support view. The revenue will be totaled though.